scripts/partition_manager.py

```diff
@@ -13,9 +13,7 @@
 
 
 def remove_item_not_in_list(list_to_remove_from, list_to_check):
-    for x in list_to_remove_from:
-        if x not in list_to_check and x != 'app':
-            list_to_remove_from.remove(x)
+    [list_to_remove_from.remove(x) for x in list_to_remove_from.copy() if x not in list_to_check and x is not 'app']
```

Comment:
IMHO it is more readable to not use a list-comprehension here. But I guess this is subjective ...